reframe/frontend/filters.py

```diff
@@ -61,7 +66,13 @@ def have_any_name(names):
     def _fn(case):
         # Check if we have an exact match
         for m in exact_matches:
-            if m == case.check.unique_name:
+            cls_name = type(case.check).__name__
+            if case.check.is_fixture():
+                fixt_id = case.check._rfm_fixt_data.mashup()
+                if (cls_name, fixt_id) == m:
+                    return True
```

Comment:
Why should we deal with fixtures here? If you select a test, its fixtures are also selected by default. And we should not allow selecting fixtures only.

Reply:
And if we need what you have here, we should not be accessing private fields of the test...

Reply:
> And we should not allow selecting fixtures only.

Before it was working so I was trying to maintain the same behavior. I think it make sense to run just the fixture when you investigate errors. If we decide to not allow it we should remove the `Rerun with ` in the failure report.

Reply:
> And if we need what you have here, we should not be accessing private fields of the test...

Yes you are right... I was trying to avoid adding another property in the pipeline, we already have `variant_num`, `param_variant` and `fixture_variant` that are different. Should add another property about data or something that only returns the `hash_id`?

Reply:
> Before it was working so I was trying to maintain the same behavior. I think it make sense to run just the fixture when you investigate errors. If we decide to not allow it we should remove the Rerun with in the failure report.

I disagree with this. Fixtures have a meaning *only* in relation to the test that are attached to, so they should not run stand alone. The fact this was allowed before it was just an artifact, never on purpose :-)